dev/com.ibm.ws.concurrent.mp.1.3_fat_tck/build.gradle

```diff
@@ -0,0 +1,12 @@
+/*******************************************************************************
+ * Copyright (c) 2022 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+
+addRequiredLibraries.dependsOn addJakartaTransformer
```

Comment:
Why does this need a Jakarta transformer?  Nothing in the 1.3 TCK should be using Java EE and in need of transformation to Jakarta EE?

Reply:
The `JakartaEE9Action` tries to load it, even if nothing needs transformed.

Reply:
Actually, I might be pointing the finger of blame in the wrong direction there, but when running with the `JakartaEE9Action` or `JakartaEE10Action` (which is used for the MP50 and MP60 repeats), it does try to use the transformer. I think from `ShrinkHelper` when an app is deployed.

Reply:
Would it be better in this case to just duplicate the server.xml for EE 10 features and have a second test class to launch the MP 6/EE 10 version of the TCK?  It seems like that would be pretty minimal and would avoid the possibility of having the transformer around to possibly contaminate things.

Reply:
Given that it should be a separate test class and app anyway, I will be removing the repeat and I'll remove this as well.

Reply:
I misread this comment yesterday as applying to the regular FAT rather than the TCK.

In the TCK, I guess we shouldn't have the same issue with the transformer. I'll re-test without it.

We could still use a separate class and server.xml to run on EE10 if necessary, but then it becomes difficult to tell which run is which in your test results.

Reply:
I tried taking the transformer library out but at the moment that causes test failures when checking for allowed FFDCs (the checker tries to transform any exception names that need it).

Stack trace:
```
java.lang.NullPointerException: 2022-08-19-12:29:02:982 null
	at componenttest.rules.repeater.EE9PackageReplacementHelper.replacePackages(EE9PackageReplacementHelper.java:72)
	at componenttest.rules.repeater.EE9PackageReplacementHelper.replacePackages(EE9PackageReplacementHelper.java:54)
	at componenttest.custom.junit.runner.FATRunner.getAllowedFFDCAnnotationFromTest(FATRunner.java:733)
	at componenttest.custom.junit.runner.FATRunner.access$400(FATRunner.java:67)
	at componenttest.custom.junit.runner.FATRunner$1.evaluate(FATRunner.java:249)
	at componenttest.rules.repeater.RepeatTests$CompositeRepeatTestActionStatement.evaluate(RepeatTests.java:143)
	at componenttest.custom.junit.runner.FATRunner$2.evaluate(FATRunner.java:352)
	at componenttest.custom.junit.runner.FATRunner.run(FATRunner.java:173)
```

I guess it's a fairly recent problem that we have tests which we want to repeat but don't need to be transformed so I'm asking around about how we should fix it.

In the mean time, we can either keep the repeat and the transformer (though the transformer will not be used to transform any apps), or use a second test class (though we'll need an extra hack in there so that you can tell the two sets of results apart).